charts/cytomine/templates/core/core-config.yaml

```diff
@@ -1,121 +1,212 @@
+        <logger name="springfox.documentation.schema.property" level="ERROR"/>
+        <logger name="sun.rmi.transport" level="WARN"/>
+
+        <logger name="org.hibernate.engine.internal.StatefulPersistenceContext" level="ERROR"/>
+        <!-->Ignore HHH000179: Narrowing proxy to class be.cytomine.domain.security.User - this operation breaks ==  ; as we have custom equals/hashcode<!-->
+
+        <!-- https://logback.qos.ch/manual/configuration.html#shutdownHook and https://jira.qos.ch/browse/LOGBACK-1090 -->
+        <shutdownHook class="ch.qos.logback.core.hook.DelayingShutdownHook"/>
+
+        <contextListener class="ch.qos.logback.classic.jul.LevelChangePropagator">
+            <resetJUL>true</resetJUL>
+        </contextListener>
+
+    </configuration>
 {{ end }}
 ---
-{{ if .Values.ims.deploy }}
+{{ if .Values.core.deploy }}
 apiVersion: v1
-kind: Secret
+kind: ConfigMap
 metadata:
-  name: {{ include "cytomine.name" . }}-ims-config
-stringData:
-  ims-config.groovy: |-
-    cytomine.ims.server.url = "http://{{ include "cytomine.name" . }}-ims:{{ .Values.ims.port }}"
-    cytomine.ims.server.core.url = "{{ include "cytomine.protocol" . }}://{{ .Values.global.domainName }}"
-    cytomine.ims.server.publicKey = "{{ $imsPub }}"
-    cytomine.ims.server.privateKey = "{{ $imsSec }}"
-
-    cytomine.ims.path.buffer = "/data/images/_buffer"
-    cytomine.ims.path.storage = "/data/images"
-
-    cytomine.ims.conversion.bioformats.enabled = {{ .Values.bioformat.deploy }}
-    cytomine.ims.conversion.bioformats.hostname = "{{ include "cytomine.name" . }}-bioformat"
-    cytomine.ims.conversion.bioformats.port = {{ .Values.bioformat.port }}
-
-    cytomine.ims.pyramidalTiff.iip.url = "http://{{ template "cytomine.name" . }}-iip-official:{{ .Values.iip_off.port }}/fcgi-bin/iipsrv.fcgi"
-    cytomine.ims.openslide.iip.url = "http://{{ template "cytomine.name" . }}-iip-cyto:{{ .Values.iip_cyto.port }}/fcgi-bin/iipsrv.fcgi"
-
-    //cytomine.ims.jpeg2000.iip.url = "http://localhost-iip-jp2000/fcgi-bin/iipsrv.fcgi"
-
-    cytomine.ims.jpeg2000.enabled = false
-
+  name: {{ include "cytomine.name" . }}-core-env
+data:
+  ADMIN_EMAIL: info@cytomine.coop
+  INSTANCE_HOST_PHONE_NUMBER: None
+  INSTANCE_HOST_SUPPORT_MAIL: None
+  REVERSE_PROXY_URL: cytomine-nginx
+  SERVER_ID: 69308206-ed7c-454b-90b3-5639377ccd2f
+  INSTANCE_HOST_WEBSITE_URL: http://cytomine.local
+  JAVAMELODY_PASS: iAWYk+Nc48XZToZ/z91R1hzhtoTgqJgxnq8mDa+I6Uo=
+  JAVAMELODY_PATH: /javamelody-core
+  JAVAMELODY_USER: admin
+  MONGO_DB_NAME: cytomine
+  MONGO_HOST: cytomine-mongodb
+  MONGO_PORT: "27017"
+  MONGO_USER: mongoadmin
+  POSTGIS_DB_NAME: docker
+  POSTGIS_HOST: cytomine-postgis
+  POSTGIS_PORT: "5432"
+  POSTGIS_USER: docker
+  SMTP_HOST: mail.smtp
+  SMTP_PASS: password
+  SMTP_PORT: "587"
+  SMTP_USER: user
+  STORAGE_PATH: /data/images
+  URL_CORE: http://cytomine.local
+  URL_PIMS1: http://ims.cytomine.local
+  URL_PIMS2: http://ims.cytomine.local
```

Comment:
Are these internal URL (should specify the service in that case, likely it will be templated), or will they be exposed to the client?

Reply:
Yes, they are internal URLs. I will template them in my next commit.